ProjFS.Mac/PrjFSKextTests/HandleOperationTests.mm

```diff
@@ -99,6 +106,19 @@ - (void) tearDown
     MockVnodes_CheckAndClear();
     MockCalls::Clear();
     MockProcess_Reset();
+
+    [super tearDown];
+}
+
+- (void) removeAllVirtualizationRoots
```

Comment:
Outside of coding style, is there any reason this helper function is written in Objective C rather than C++?  (In other words, is this required for asserts to work as expected?)

Reply:
It references ivars in the test class (I've made this explicit via `self->`, which is technically not required) so we'd need to pass pointers/references to those ivars as arguments if we implemented this as a free function. I didn't see an advantage in doing that. The decision has nothing to do with asserts though.